src/routes/patterns/components/Buttons.js

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import { Button, ButtonGroup, Card } from 'components/baseline';
 
 const Buttons = () => (
-  <Card>
+  <Card className='Buttons'>
```

Comment:
we should get consistent about quotes on html attributes. They should all have double quotes. Maybe adding eslint rules would help